faasmcli/faasmcli/tasks/cluster.py

```diff
@@ -18,7 +19,10 @@ def start(ctx, workers=2, sgx=FAASM_SGX_MODE_DISABLED):
     """
     Start the local dev cluster
     """
+    # This env makes sure we mount our local setup into the containers, rather
+    # than using the prebuilt binaries
     env = copy(os.environ)
+    env["FAASM_BUILD_DIR"] = join(PROJ_ROOT, "dev/faasm/build")
```

Comment:
`FAAASSM_BUILD_DIR` would get set by `bin/workon.sh` which would pick up the native build dir, not this one.